### DISCUSSION TITLE
HvDropdown in combination with useContext() issue

### DISCUSSION THREAD

**Discussion by pavvarpet** [lumada-design/hv-uikit-react#4199] [Q&A]
Greetings, and thank you for your work.

Is it possible to use dropdown onChange method to set the value inside react state?

When we did something like:
```
  const onChange = (e: any) => {
    console.log(e);
    console.log("Implement this onChange");
    setVal(e.label);
    console.log(val + " is selected.");
  };
  const [val, setVal] = useState("");
  ```
  
In this case the dropdown component becomes broken for some reason... :/

**Comment by zettca** [comment]
Closing this as it's being addressed in https://github.com/lumada-design/hv-uikit-react/issues/4202
